packages/kind2/kind2.1.4.0/opam

```diff
@@ -24,7 +24,7 @@ depends: [
   "ounit2" {with-test}
   "z3" {< "4.8.9" & with-test}
   "yojson"
-  "zmq" {>= "5.0.0"}
```

Comment:
Why was 5 wrong?

Reply:
zmq 5.0.0 requires ocaml < 4.06.0 (zmq → configurator < v0.15 → base < v0.14 → ocaml < 4.06.0). All versions of Kind 2 requires at least ocaml 4.07, so zmq 5.1.0 is the smallest version I could install and use to build Kind 2.

Reply:
Thanks